fmriprep/cli/run.py

```diff
@@ -108,11 +108,13 @@ def main():
             sentry_sdk.capture_message(success_message, level='info')
 
         # Bother users with the boilerplate only iff the workflow went okay.
-        if (config.execution.output_dir / 'fmriprep' / 'logs' / 'CITATION.md').exists():
+        _boiler_file = config.execution.output_dir / "fmriprep" / "logs" / "CITATION.md"
+        if _boiler_file.exists():
```

Comment:
We're scoped inside a function. Private variables don't mean anything.

```suggestion
        boiler_file = config.execution.output_dir / "fmriprep" / "logs" / "CITATION.md"
        if boiler_file.exists():
```